clar2wasm/tests/wasm-generation/control_flow.rs

```diff
@@ -0,0 +1,40 @@
+use clar2wasm::tools::crosscheck_compare_only;
+use clarity::vm::types::TypeSignature;
+use proptest::proptest;
+use proptest::strategy::Strategy;
+
+use crate::{prop_signature, PropValue};
+
+fn is_optional_type(ty: TypeSignature) -> bool {
+    matches!(ty, TypeSignature::OptionalType(_))
+}
+
+proptest! {
+    #![proptest_config(super::runtime_config())]
+
+    #[test]
+    fn crossprop_unwrap_panic(
+        vals in prop_signature().
+        prop_filter("filter", |ty| is_optional_type(ty.clone()) || ty.is_response_type()).
```

Comment:
That is **VERY** expensive and inefficient. The documentation basically [tells you to nut use it](https://docs.rs/proptest/latest/proptest/strategy/trait.Strategy.html#method.prop_filter).
Especially here, you will have so many rejects.

Reply:
Ok. Thanks for the tip. Changed.